lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -188,6 +188,13 @@ public function get_item_permissions_check( $request ) {
 
 		$id = (int) $request['id'];
 		$user = get_userdata( $id );
+		$types = array();
+		foreach ( get_post_types( array(), 'object' ) as $obj ) {
```

Comment:
Can't we just do `$types = get_post_types( array( 'show_in_rest' => true ), 'name' )`